documentation/modules/ROOT/pages/operations/openshift.adoc

```diff
@@ -8,56 +8,75 @@
 
 toc::[]
 
-The following describes how to set up the {prodname} connectors for change data capture on Red Hat's https://www.openshift.com/[OpenShift] container platform.
+This procedure is for setting up {prodname} connectors on Red Hat's link:https://www.openshift.com/[OpenShift] container platform. These instructions have been tested with the two most recent releases of OpenShift. 
 
-These instructions have been tested using the https://github.com/minishift/minishift[Minishift tool]
--- allowing you to run a single node OpenShift instance locally on your machine.
+To get started more quickly, try the link:https://learn.openshift.com/middleware/debezium-getting-started/[{prodname} online learning scenario].
+It starts an OpenShift cluster just for you, which lets you start using {prodname} in your browser within a few minutes.
 
-You can find a complete example of this set-up using Minishift in our https://github.com/debezium/debezium-examples/tree/master/openshift[examples repository].
+== {prodname} Deployment
 
-And if you want to get started even quicker, try out the https://learn.openshift.com/middleware/debezium-getting-started/[{prodname} online learning scenario].
-It starts an OpenShift cluster just for you, allowing you to take your first steps with {prodname} in your browser just within a few minutes.
+To set up Kafka and Kafka Connect on OpenShift, use the set of images that are provided by the link:https://strimzi.io/[Strimzi] project. These images offer "Kafka as a Service" by providing enterprise grade configuration files and images that bring Kafka to OpenShift.
```

Comment:
```suggestion
To set up Apache Kafka and Kafka Connect on OpenShift, use the set of images that are provided by the link:https://strimzi.io/[Strimzi] project. These images offer "Kafka as a Service" by providing enterprise grade configuration files and images that bring Kafka to Kubernetes and OpenShift, as well as Kubernetes operators for running Kafka there.
```

Reply:
I updated it as you suggested.

Reply:
Cool. Btw. you can also directly suggestions like mine here in the UI (see the two buttons above).